src/app/desktop/utils/index.js

```diff
@@ -293,7 +303,7 @@ export const isLatestJavaDownloaded = async (meta, userData, retry) => {
         }
       }
 
-      return isLatestJavaDownloaded(meta, userData);
+      return isLatestJavaDownloaded(meta, userData, null, version);
```

Comment:
why no retry?

Reply:
Then we would create an infinite recall loop.

Reply:
Then we would create an infinite recall loop